Tasks/PackerBuild/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+        "target": "ES6",
+        "module": "commonjs",
+        "sourceMap": true,
+        "sourceRoot": "E:\\github\\vsts-tasks\\Tasks\\PackerBuild" 
```

Comment:
on further thinking, may be I will remove this - it is better to have every ennggr his own way